sdks/java/container/license_scripts/pull_licenses_java.py

```diff
@@ -35,18 +35,9 @@
 from tenacity import retry
 from tenacity import stop_after_attempt
 from tenacity import wait_exponential
+from urllib.request import urlopen, URLError, HTTPError
+
 
-try:
-    # py2
-    from future.moves.urllib.request import urlopen
-    from future.moves.urllib.request import URLError, HTTPError
-except:
-    # py3
-    from future import standard_library
-    from urllib.request import urlopen, URLError, HTTPError
```

Comment:
@Hannah-Jiang @udim should we keep the py2 imports for now? I still have python 2 as the default `python` on my machine, so this change broke :sdks:java:container:docker for me

Or maybe I should have changed the default by now?

Reply:
This is running in a virtualenv as implemented [here](https://github.com/apache/beam/blob/master/sdks/java/container/license_scripts/license_script.sh#L28).

When I check locally with py2, it created py3.7 env. Default Python version on Jenkins is also 2.7 and the script run successfully. Maybe in both cases, Python3 was installed on the machine, so it is able to create virtualenv with Python3?

Can you please check what version of Python was installed at sdks/java/container/build/virtualenv?

Reply:
Does  adding `--python=python3` to the virtualenv command fix this for you, Brian?

Reply:
> Can you please check what version of Python was installed at sdks/java/container/build/virtualenv?
```
❯ ./sdks/java/container/build/virtualenv/bin/python --version
Python 2.7.18rc1
```

If this is working for others maybe I just have something else wrong with my setup...

> Does adding --python=python3 to the virtualenv command fix this for you, Brian?
yeah that's what I did to get it passing. I thought about putting up a PR for the change, but not sure if that's a universal solution or not.

Reply:
I think your setup is fine - it defaults to Python 2. Since the `from urllib.request import urlopen, URLError, HTTPErro` line doesn't work on 2.x, adding the flag should be a universal solution.

Reply:
Well what I'm wondering is if `--python=python3` always works, or if it only works if you have a `python3` symlink, etc...

Reply:
Yes, we use it elsewhere in our build (BeamModulePlugin.groovy)